pennylane_lightning/core/src/utils/UtilLinearAlg.hpp

```diff
@@ -15,69 +15,31 @@
 
 /**
  * @file
- * LAPACK wrapper functions declarations.
+ * BLAS wrapper functions declarations.
  */
 #pragma once
 
 #include <algorithm>
-#include <array>
 #include <complex>
-#include <cstdlib>
-#include <filesystem>
-#include <memory>
 #include <string>
 #include <vector>
 
-#include <iostream>
-
-#include "SharedLibLoader.hpp"
+#include "BLASLibLoaderManager.hpp"
 
 /// @cond DEV
 namespace {
 // Declare heev function pointers to access corresponding functions in
-// LAPACK/OpenBLAS
+// OpenBLAS
 using zheevPtr = void (*)(const char *, const char *, const int *,
                           std::complex<double> *, const int *, double *,
                           std::complex<double> *, const int *, double *, int *);
 using cheevPtr = void (*)(const char *, const char *, const int *,
                           std::complex<float> *, const int *, float *,
                           std::complex<float> *, const int *, float *, int *);
```

Comment:
Do we still need these function pointer declarations if the lib is being pulled in directory?